packages/type-safe-api/docs/developer_guides/type-safe-api/index.md

```diff
@@ -653,24 +679,29 @@ Add the new operation in the `model` project, for example:
                     - message
     ```
 
-    Note that you may wish to split your API into multiple `yaml` files. Please see [Using OpenAPI](using_openapi.md) for details.
+    Note: You can split your API into multiple `yaml` files. For more information, refer to [Using OpenAPI](using_openapi.md).
 
-### Build your Project
+### Build your project
 
-Run a build in the root of your monorepo using your package manager's build command, eg `yarn build`.
+To run a build in the root of your monorepo, use your package manager's build command.
 
-The build will regenerate the infrastructure, runtime, documentation and library projects based on your updated model.
+```
+yarn build
+```
 
-You'll most likely get a build error in your CDK application now! For example:
+The build will regenerate the infrastructure, runtime, documentation, and library projects based on your updated model.
 
-```
+You may see the following build error in your CDK application.
+
+```ts
 TSError: ⨯ Unable to compile TypeScript:
-src/main.ts(16,7): error TS2741: Property 'sayGoodbye' is missing in type '{ sayHello: { integration: Integration; }; }' but required in type 'OperationConfig<TypeSafeApiIntegration>'.
+src/main.ts(16,7): error TS2741: Property 'sayGoodbye' is missing in type '{ sayHello: { integration: Integration; }; }' but required in type 'Operation
+Config<TypeSafeApiIntegration>'.
 ```
 
-This is because we need to define an integration for every operation in our API. Let's do this next!
+To troubleshoot this, we need to define an integration for every operation in our API.
```

Comment:
I'm not sure "to troubleshoot this" makes sense here - the error is expected because the CDK construct requires an integration for every operation, and we just added a new operation which we haven't added an integration for yet :)

Maybe it makes more sense to say:

> Since an integration must be defined for every operation, you may see a build error in your CDK application.

{{error}}

> This is expected, and can be resolved by continuing with the steps below."

Reply:
updated